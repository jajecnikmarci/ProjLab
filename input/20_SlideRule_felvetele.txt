starttest
add room room1 3
add item room1 SlideRule slide
add person room1 Student student1
printstate room1
printstate student1
pickup student1