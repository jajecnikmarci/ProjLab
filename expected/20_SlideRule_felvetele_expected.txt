room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
		slide
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: 
		StickinessEffect
student1
	- location: room1
	- stunned: false
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
JATEK VEGE, hallgatok nyertek.
